x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/saml/SamlRealmTests.java

```diff
@@ -123,19 +123,20 @@ public void testReadIdpMetadataFromHttps() throws Exception {
         final Path path = getDataPath("idp1.xml");
         final String body = new String(Files.readAllBytes(path), StandardCharsets.UTF_8);
         final MockSecureSettings mockSecureSettings = new MockSecureSettings();
-        mockSecureSettings.setString("xpack.ssl.secure_key_passphrase", "testnode");
+        mockSecureSettings.setString("xpack.security.transport.ssl.secure_key_passphrase", "testnode");
```

Comment:
Doesn't really affect anything but we could use `xpack.security.http` key since the produced SSLContext is used in a MockWebServer